docs/docs/migration/airflow-to-dagster/airflow-operator-migration/bash-operator-dbt.md

```diff
@@ -4,6 +4,10 @@ sidebar_position: 200
 title: Migrating an Airflow BashOperator (dbt) to Dagster
 ---
 
+import UseAirliftComponent from '@site/docs/partials/\_UseAirliftComponent.md';
+
+<UseAirliftComponent />
+
```

Comment:
the destination ([here](https://nikki-docs-integrations-update.archive.dagster-docs.io/guides/build/components/integrations/airflow-component-tutorial)) doesnt have the relevant dbt info tho. i wonder if this could confuse people.

Reply:
Good point -- I assume all the operators in https://docs.dagster.io/migration/airflow-to-dagster/airflow-operator-migration/ are implemented in the component version of Airlift? If so, I can move those reference docs. cc @dpeng817